### PR TITLE
Add a default dummy entry for the stack dropdown

### PR DESCRIPTION
**Pull request by kiwiz**

### CODE REVIEW

File: lib/deployinator/templates/layout.mustache [hunk 1]
```diff
@@ -53,6 +53,7 @@ _  __  / _  _ \___  __ \__  / _  __ \__  / / /__  / __  __ \_  __ `/_  __/_  __
                        <span class="label">Stack: </span>
                        <span>
                            <select name='stacks' id='stacks'>
+                           <option></option>
```
Comment (jayson) [inline lib/deployinator/templates/layout.mustache]: Perhaps we should put some default text in here as well, like "Choose a stack" or just "Choose..."

**Comment by kiwiz** [comment]
Done. :+1:

**Comment by jayson** [comment]
Awesome thanks! Merging

**Comment by kiwiz** [comment]
:rocket:

**Pull request merged**